wadm/lib/config_plan.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Wadm.ConfigPlan do
+  use Vapor.Planner
+  dotenv()
+
+  config :nats,
+         env([
+           {:host, "NATS_HOST", default: '172.18.0.1'},
```

Comment:
According to a quick google, the use of this address often allows for IP tunneling vulnerabilities. Can we default to `127.0.0.1` instead @janitha09  ?

Reply:
fixed